AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -8,24 +8,34 @@
 import SwiftUI
 
 struct SessionHeaderView: View {
-    
+    func placeOrder() {}
+    func adjustOrder() {}
+    func cancelOrder() {}
```

Comment:
I guess they're for the future but I wouldn't put them here if there's no need for that (aka you don't use empty funcs for example for buttons). 
And I don't understand - what are they going to do? The buttons say resume recording, edit, share or delete and those funcs names are different :D